testsuite/cluster-test/src/effects/reboot_validator.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+/// RebootValidator introduces a rebooting for a given instance
+use crate::{effects::Effect, instance::Instance};
+use anyhow::Result;
+
+use async_trait::async_trait;
+use libra_logger::info;
+use std::fmt;
+
+pub struct RebootValidator {
```

Comment:
Seems like it should be named `StopValidator`

Reply:
make sense, will update the name